edgy/core/db/context_vars.py

```diff
@@ -1,11 +1,13 @@
 from contextvars import ContextVar
-from typing import TYPE_CHECKING, Type, Union
+from typing import TYPE_CHECKING, Union
 
 if TYPE_CHECKING:
-    from edgy import Database, Model, QuerySet
+    pass
```

Comment:
This can be removed since nothing is inside the check

Reply:
good idea